### PR TITLE
Formatted all pages according to guidelines.

### PR DESCRIPTION
**Pull request by rubenvereecken**
Alright, don't merge this in yet. I will after due discussion.

All pages are now formatted according to discussion in #622. It's not final, as not everyone has had their say yet.

It's important that once this gets merged in we update `CONTRIBUTING.md`, and be very vigilant when merging in new changes. At least until I get time to incorporate my linter. It's best to have it start with a clean slate though, so Travis can play with it after every PR.

Summary of changes: Whitespace, top descriptions are full sentences and example description are sentences ending in a colon. The last is still under debate.

Hurray [`tldr-lint`](https://github.com/rubenvereecken/tldr-lint) and its new brother, `tldr-format`. Soon available through `npm`.

### CODE REVIEW

**Comment by igorshubovych** [comment]
Oh my. What a commit!?!

I hope @waldyrious will not be captious to it, otherwise we will never merge it.

**Comment by igorshubovych** [comment]
TL;DR. LGTM. :)

**Comment by rubenvereecken** [comment]
Alright I'll leave this out for a bit for @waldyrious, @ostera or @felixonmars to have a look if they wish. Merging tomorrow.

**Comment by igorshubovych** [comment]
Oops. We have to merge it ASAP. Otherwise such a huge commit will diverge a lot.

**Comment by ostera** [comment]
Okay so if we take the time to do this we should stop merging PR's for at least a day. Go through this change, merge it, and then continue with the rest of the PR's.

It has my upvote to stop merging PR's and start looking into it.

**Comment by igorshubovych** [comment]
@ostera :+1:

**Comment by waldyrious** [comment]
I pledge to not merge anything until this gets merged, too :)

As for reviewing the PR... I'll take @rubenvereecken's word on this one -- have a life, you know :P

Kidding aside: I do see some things I could point out, but they're not issues with this particular PR, but with the pages themselves (i.e. using the gerund rather than infinitive). So from my side, there's clearance to merge :+1:

**Comment by rubenvereecken** [comment]
Rebased and cleaned, I'll merge this in now. 

I'm off for the weekend and will be back on Tuesday. Next up in the works is `tldr-lint`. As soon as that's finished I'll halt everything again, apply `tldr-format` to whatever managed to sneak by and get Travis set up to do the work for us in the future.

**Comment by waldyrious** [comment]
Awesome @rubenvereecken :) +:100:

**Comment by ostera** [comment]
:100:  :smile:
